homeassistant/components/elkm1/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "elkm1",
   "name": "Elk-M1 Control",
   "documentation": "https://www.home-assistant.io/integrations/elkm1",
-  "requirements": ["elkm1-lib==0.8.8"],
+  "requirements": ["elkm1-lib==0.8.10"],
```

Comment:
The version in the library source doesn't seem to have been changed?

https://github.com/gwww/elkm1/blob/7572df6b20f044816bca05ed503c516f5f606a2a/pyproject.toml#L3

Reply:
My mistake. Fixed.